src/workerinfo.js

```diff
@@ -34,57 +35,65 @@ class WorkerInfo {
     }
   }
 
-  async seen(provisionerId, workerType) {
+  async seen(provisionerId, workerType, workerGroup, workerId) {
     const expired = entity => Date.now() - new Date(entity.expires) > 24 * 60 * 60 * 1000;
     const expires = taskcluster.fromNow('5 days');  // temporary hard coded expiration
```

Comment:
Now that I think about this some more, there are a few things that will prevent that:
* expiration only runs daily
* expiration is only updated in the table every 6 hours

So I think the minimum expiration is one day.

In fact, looking at
```js
 const expired = entity => Date.now() - new Date(entity.expires) > 24 * 60 * 60 * 1000;
```
I think we wait until 24 hours *after* an entry is expired to write its new expiration date?  Shouldn't that be the reverse?